spec/lib/booking_bug/api_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
```

Comment:
We shouldn't require `rails_helper` to test out library code. Our library should not depend on Rails, and thus should be testable with just `spec_helper` that is loaded by default.